scripts/sdk_installers/install_sp1_sdk.sh

```diff
@@ -29,12 +29,14 @@ curl -L https://sp1up.succinct.xyz | bash -s -- --yes
 # and for subsequent commands if this script is sourced.
 export PATH="${SP1UP_HOME}/bin:${SP1_HOME}/bin:$PATH"
 
+export SDK_VERSION="${SP1UP_VERSION:-latest}"
```

Comment:
Since this script is somewhat general, if the env var isn't provided we download the latest release as before.